SocialNetwork/SocialNetwork.Core/Services/UserService.cs

```diff
@@ -1,36 +1,63 @@
 ﻿using SocialNetwork.Core.Interfaces;
 using SocialNetwork.Core.Models;
-
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
 namespace SocialNetwork.Core.Services;
-
 public class UserService : IUserService
 {
-    private readonly IRepository _repository;
```

Comment:
There should be interface IRepository to work with it. Wait for this interface to appear in develop and then merge changes to this branch